doc/input-output-reference/src/overview/group-electric-load-center-generator.tex

```diff
@@ -1184,7 +1184,7 @@ \subsubsection{Inputs}
 
 \paragraph{Field: Maximum Module Capacity}\label{field-maximum-module-capacity}
 
-This field indicates the maximum capacity of one battery module. It is evaluated as the total Amp-hours available when a full battery is discharged at infinitesimal current. This capacity is for an individual battery module, not for the whole battery bank. The maximum capacity can be found from manufacture s data or derived from test data by curve fitting.
+This field indicates the maximum capacity of one battery module. It is evaluated as the total Amp-hours available when a full battery is discharged at infinitesimal current. This capacity is for an individual battery module, not for the whole battery bank. The maximum capacity can be found from manufacture's data or derived from test data by curve fitting.
```

Comment:
I'm not sure `manufacture's` is better than `manufacture s`, but I"m not interested in holding up the PR for more commits.